content/blog/2019-12-03-mark-padgham.md

```diff
@@ -0,0 +1,76 @@
+
+
+### You’ve made so many concrete contributions to rOpenSci, as well as less-visible community contributions. What made you keep coming back to give more?
+
+It comes back to my belief in software as a tool to make the world a better place. Part of that is an active effort on the part of myself and the others with whom I work and develop packages to develop a comprehensive ecosystem. I make an active personal effort to develop a comprehensive ecosystem of packages that presents an open source alternative to closed source commercial environments. And that open source alternative will hopefully help to openly transform the world.
+
+It’s in my own interest to continue to contribute to the community because it’s only in that way that what I have contributed continues to maintain its value. Without ongoing contributions to rOpenSci that whole thing stops at the point at which I left it and all I’ve got is to sit back and hope something happens. That’s worth nothing in comparison to me continuing to actively strive to foster, cultivate, grow, expand and enrich this ecosystem of tools in our little niche.
+
+
+### What will you be working on and what do you think are some of the key challenges?
+
+I’ll be working on developing robust tools to assess packages. As an inveterate coder, I love nothing more than to get up in the morning and find myself still sitting there at 4:00 in the afternoon in my “coding-hole”. But a big part of the initial stuff, which I’m really excited about, is about connecting various people who will ultimately be involved in guiding the expansion of rOpenSci into the statistical methods side of things. So it’s much more people-centered than the typical life of a 100% coder. Like so many rOpenSci projects, this aims to be very community-driven.
+
+I’ll really enjoy being between two worlds trying to think in general terms about developing this whole new endeavour [statistical software peer review] and ensuring the development is as general and generalizable as possible. While on the other side, having to be involved with profoundly important brilliant academic experts in various aspects of statistics & statistical software so it will be both very niche and very general at the same time. One of the key challenges will be working with experts in the statistical and software communities and having to foster consensus of opinion on their part in realms that have basically never been tried before. And on the other side, I’ve got a lot of concrete visions for extending testing well beyond anything that’s been done in a computational or quantitative way. Convincing leaders in statistics and statistical software that’s a good idea is certainly going to be challenging and very rewarding.
```

Comment:
"I’ve got a lot of concrete visions for extending testing ~~well~~ beyond ~~anything that’s been done in a~~ current computational or quantitative ~~way~~approaches. Convincing leaders in statistics and statistical software ~~that’s a good idea~~to update and innovate on current practices is certainly going to be challenging and very rewarding." [updated to reflect the above quote]